spec/features/vue_playground_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe 'Vue Playground', :prerendering_disabled do
```

Comment:
The specific examples that need prerendering disabled have it marked below. We don't need to mark it up here, and, by removing this, there is now one spec we run with prerendering enabled where it wasn't before, which I think is a small win (diverging less from production).